build.gradle

```diff
@@ -25,8 +26,9 @@ apply plugin: 'io.codearte.nexus-staging'
 description = 'JGiven - BDD in plain Java'
 def jacocoEnabled = !JavaVersion.current().isJava11Compatible();
 
-task wrapper(type: Wrapper) {
-    gradleVersion = '4.10.2'
+wrapper {
+    gradleVersion = '5.1'
```

Comment:
Gradle `5.1.1` is the latest version and contains a few bug fixes. I suggest you use this version.